examples/gadgets/basic/trace/dns/README.md

```diff
@@ -23,28 +23,19 @@ Then, perform some DNS requests:
 
 
 ```bash
-$ nslookup microsoft.com
-$ nslookup google.com
+$ nslookup -querytype=a microsoft.com.
+$ nslookup -querytype=a google.com.
```

Comment:
Without the final dot, it will try all search options:

```
$ grep search /etc/resolv.conf 
search default.svc.cluster.local svc.cluster.local cluster.local
```
So it would search
* microsoft.com.default.svc.cluster.local.
* microsoft.com.svc.cluster.local.
* microsoft.com.cluster.local.

I think it clutters the documentation.

Reply:
Agree